src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -53,65 +56,23 @@
 public class IEEEXploreFetcher implements EntryFetcher {
 
     private static final Log LOGGER = LogFactory.getLog(IEEEXploreFetcher.class);
+    private static final String URL_SEARCH = "http://ieeexplore.ieee.org/rest/search?reload=true";
+    private static final String URL_BIBTEX_START = "http://ieeexplore.ieee.org/xpl/downloadCitations?reload=true&recordIds=";
+    private static final String URL_BIBTEX_END = "&download-format=download-bibtex&x=0&y=0";
+    private static final String DIALOG_TITLE = Localization.lang("Search %0", "IEEEXplore");
+    private static final int MAX_FETCH = 100;
 
     final CaseKeeperList caseKeeperList = new CaseKeeperList();
```

Comment:
Not sure, existing code. But I can't find it anywhere else in the project, so I removed it.